content/articles/how-to-ground-truth-label-your-datasets/index.md

```diff
@@ -0,0 +1,163 @@
+
+![next frame](/engineering-education/how-to-ground-truth-label-your-datasets/ground10.png)
+
+We notice that we can go ahead and make the labeling again, but this will be tiresome and boring since it is a repetition of what we did before. It is where automation comes in.
+
+### Use an algorithm to automate the labeling process.
+There are so many in-built algorithms in Matlab that help you with this. You can use an in-built algorithm or write your algorithm to do this; Matlab accepts all these options. 
+
+To use an algorithm, click on the `select algorithm` and select or import algorithm to use.
+
+![algorithms](/engineering-education/how-to-ground-truth-label-your-datasets/ground11.png)
+
+For this tutorial, we will use the `Point Tracker`. After selecting the algorithm, you select the vehicles you want to track. 
+
+Also, you input how long you want to track it. It is specified beside the play section where you input the `start time` and `end time`. 
+
+For example, we are going to track the cars for seven seconds.
+
+![tracking](/engineering-education/how-to-ground-truth-label-your-datasets/ground12.png)
+
+After this, you click on `automate`. Once you click this, the window displays the instruction for using this algorithm. Read through these instructions and then click on the play at the top of the window.
+
+![automation](/engineering-education/how-to-ground-truth-label-your-datasets/ground12a.png)
+
+Once it completes, click on the `accept`. You can look at the `label summary` to see how the labeling proceeded.
+
+![label summary](/engineering-education/how-to-ground-truth-label-your-datasets/ground13.png)
+
+### Other capabilities
+Let's say that we want to label the lanes. In this case, we cannot use bounding boxes; we will use the line and label the lanes. It means that you draw lines on the lanes, and they will be labeled.
+
+![labeling the lane](/engineering-education/how-to-ground-truth-label-your-datasets/ground14.png)
+
+Also, you can label individual pixels in the image using the `pixel label`. For example; we want to label the road, so we will make a label road and use the pixel label.
+
+![using pixel label](/engineering-education/how-to-ground-truth-label-your-datasets/ground15.png)
+
+Note that you can erase sections that you never intended to be labeled using `erase`, and they also include those that you may have missed using the `brush`. We also have a feature, `smart polygon`. This feature allows you to make a surrounding of what you want to label and makes its segmentation. After all these, you can save your work.
+
+### Integrate ground truth information from other sensors.
+This feature allows you to integrate ground truth information from other signals. For example, let's open a video that has been time-synchronized with lidar data. 
+
+To open this, execute the command below:
+
+```Matlab
+groundTruthLabeler('01_city_c2s_fcw_10s.mp4', 'ConnectorTargetHandle', @LidarDisplay);
```

Comment:
Where is this file mentioned?

Reply:
@linetachieng is the video file provided by Matlab?